source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/with-project-operations.tsx

```diff
@@ -37,11 +37,7 @@ export function withProjectOperations(WrappedComponent) {
         );
       } catch (e) {
         console.error(e);
-        toast.error(
-          !dateArchived ?
-            t('project.operations.archive.error') :
-            t('project.operations.reactivate.error')
-        );
+        toast.error(e);
```

Comment:
does toast.error know how to handle `Error` objects?

Reply:
it does. Most likely because it calls toString of Error object